platform/node/src/node_expression.cpp

```diff
@@ -157,8 +160,8 @@ struct ToValue {
     v8::Local<v8::Value> operator()(const std::vector<Value>& array) {
         Nan::EscapableHandleScope scope;
         v8::Local<v8::Array> result = Nan::New<v8::Array>();
-        for (std::size_t i = 0; i < array.size(); i++) {
-            result->Set(i, toJS(array[i]));
+        for (std::uint32_t i = 0; i < array.size(); i++) {
```

Comment:
Note that static analysers would still find this change offensive as `size_t` can be 64 bit, but it's fine for now.